frontends/web/src/locales/en/app.json

```diff
@@ -833,13 +833,33 @@
       }
     },
     "settings-electrum": {
+      "connection": {
+        "text": "If you  intend to only connect to your node when you are on the same network (e.g. your home wifi), then using regular network communication is sufficient.\nBe aware that your Electrum server needs to provide a TLS certificate to encrypt the communication in that case.\nIf you intend to connect to your node from anywhere, using Tor is the better option. No TLS certificate is necessary in that case.",
```

Comment:
> If you  intend to only connect to your node when you are on the same network (e.g. your home wifi), then using regular network communication is sufficient.

what does that mean? regular network communication?

> Be aware that your Electrum server needs to provide a TLS certificate to encrypt the communication in that case.

It is advisable, but not required.

Reply:
Re clearnet: Did that change recently? I can swear that the node needs to provide a TSL certificate if you try to connect via Clearnet.

Reply:
I proposed the "regular network connection" to avoid "clearnet", which is basically a word made up by us... Any other non-technical description for regular TCP/IP communication is welcome. :)

Re: clearnet TLS, I was also under the impression that we only allow skipping TLS over Tor.

Reply:
>  Did that change recently?

Since 4.19.0, you can use TCP only (no TLS), regardless of Tor.

Before that, TLS was required in all cases, even for Tor.

Reply:
double space

Reply:
removed double space, updated "required" --> "advisable".